### PR TITLE
Add `is.formData` and `is.urlSearchParams`

### PR DESCRIPTION
**Pull request by kaysonwu**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you also add it to the readme?

**Comment by kaysonwu** [comment]
@sindresorhus okay!

**Comment by sindresorhus** [comment]
https://github.com/kaysonwu/is/commit/26d668c9f1bc6e047ef2903fdbe01c89b2d7da6f ?

**Comment by kaysonwu** [comment]
`URLSearchParams` and `FormData` are usually used together.

File: readme.md [hunk 1]
```diff
@@ -432,6 +432,28 @@ Returns `true` if `value` is an odd integer.
...
+is.formData(data);
+//=> true
+```
+
+##### .urlSearchParams(value)
+
+Returns `true` if `value` is an instance of the [`URLSearchParams` class](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams).
+
+```js
+const data = new URLSearchParams();
```
Comment (sindresorhus) [inline readme.md:451]: Incorrect variable name

File: test/test.ts [hunk 2]
```diff
@@ -1621,6 +1621,48 @@ test('is.all', t => {
...
+	t.throws(() => {
+		assert.formData(undefined);
+	});
+	t.throws(() => {
+		assert.formData(null);
+	});
+});
+
+test('is.urlSearchParams', t => {
+	const params = new window.URLSearchParams();
```
Comment (sindresorhus) [inline test/test.ts:1646]:
```suggestion
	const searchParameters = new URLSearchParams();
```

`URLSearchParams` is a global in Node.js

**Pull request merged**